### DISCUSSION TITLE
ESAMI DI STATO

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#400] [Richieste di aiuto] [answered]
Buonasera, una domanda: nel registro è presente la gestione degli Esami di Stato? ricordo, ad esempio, che in lampschool fosse possibile anche gestire le commissioni e le valutazioni finali degli esami (prove scritte, orale, etc).
Grazie.

**Comment by trinko** [comment, marked as answer]
No, gli Esami di Stato non vengono gestiti. 

Poiché il registro è pensato per le scuole superiori, la gestione che indichi viene fatta sull'applicativo del MIUR, Commissione Web.
